fineract-provider/src/main/java/org/apache/fineract/infrastructure/security/utils/SQLBuilder.java

```diff
@@ -122,6 +122,18 @@ public String getSQLTemplate() {
         return "";
     }
 
+    /**
+     * Returns the added criterias, created from the {@link #addCriteria(String, Object)}, with '?' placeholders.
+     *
+     * @return added criterias
+     */
+    public String getCriteria() {
+        if (sb.length() > 0) {
+            return sb.toString();
+        }
+        return "";
+    }
+
```

Comment:
I thought we would need this, initially, since the "where" was already written in a SQL query which I was not able to replace with sqlBuilder. Please look into other comments, if solved I think this would resolve (needed/to be removed) itself :)